perun-core/src/main/java/cz/metacentrum/perun/core/implApi/GroupsManagerImplApi.java

```diff
@@ -158,6 +158,16 @@
 	 */
 	Group getGroupByName(PerunSession perunSession, Vo vo, String name) throws GroupNotExistsException;
 
+	/**
+	 * Search for the groups with specified ids in all VOs.
```

Comment:
I would prefer having similar messages in javadoc for all those impl methods. Its ``get`` method. It doesn't search/find. Also you should note, that it silently skips non-existing entities.

Reply:
Done